nixos/modules/services/accessibility/orca.nix

```diff
@@ -0,0 +1,25 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.orca;
+  inherit (lib)
+    mkEnableOption
+    mkIf
+    mkPackageOption
+    ;
+in
+{
+  options.services.orca = {
+    enable = mkEnableOption "Orca screen reader";
+    package = mkPackageOption pkgs "orca" { };
```

Comment:
Not sure if we need package option. Orca should not really need overriding for general use, and in case of e.g. debugging, as a leaf package, it can be easily overridden with overlays.

Reply:
I do not see the point but it’s really a minor thing.